scout/build/individual.py

```diff
@@ -122,6 +122,9 @@ def build_individual(ind: dict) -> dict:
     if "msi" in ind:
         ind_obj["msi"] = ind["msi"]
 
+    if "hrd" in ind:
```

Comment:
Looks good! The complexity warning is sure not your fault. If you want, you might easily reduce the complexity with a loop like 
```suggestion
    for optional_ind_key in ["hrd", "msi","tmb", "tumor_purity", "tumor"]:
        if optional_ind_key in ind:
            ind_obj[optional_ind_key] = ind[optional_ind_key]
```
The same goes for most assignments here. Nowadays we would have made it a Pydantic class among the models and instantiated it here to solve the same thing, but don't feel you have to just to fix this one thing: its as readable as before.